js/global.js

```diff
@@ -2102,20 +2125,65 @@ if (typeof String.prototype.trim !== 'function') {
     };
 }
 
-// jQuery UI .effect() replacement using CSS classes.
-jQuery.fn.effect = function(name) {
-    var that = this;
-    name = name + '-effect';
+jQuery.fn.extend({
+    // jQuery UI .effect() replacement using CSS classes.
+    effect: function(name) {
+        alert('effect');
+        var that = this;
+        name = name + '-effect';
+
+        return this
+            .addClass(name)
+            .one('animationend webkitAnimationEnd', function () {
+                that.removeClass(name);
+            });
+    },
+
+    accessibleFlyoutHandle: function (isOpen) {
+        $(this).attr('aria-expanded', isOpen.toString());
+    },
+
+    accessibleFlyout: function (isOpen) {
+        $(this).attr('aria-hidden', (!isOpen).toString());
+    },
+
+    setFlyoutAttributes: function () {
+        var $wrapper = $(this);
```

Comment:
This should probably have an `each()` to handle multiple flyouts being selected.